community/cypher/cypher/src/test/scala/org/neo4j/cypher/ProfilerAcceptanceTest.scala

```diff
@@ -47,6 +47,16 @@ class ProfilerAcceptanceTest extends ExecutionEngineFunSuite with CreateTempFile
     assertDbHits(2)(result)("Expand")
   }
 
+  test("match n where n-[:FOO]->() return * 222") {
+    //GIVEN
+    relate( createNode(), createNode(), "FOO")
+
+    //WHEN
+    val result = eengine.execute("planner cost match n return n").toList
+    println(result)
+
+  }
+
```

Comment:
Accidentially added to commit? Doesn't appear to make much sense, in this commit, or as a proper test ...